Documentation/Contributing/Roadmap.md

```diff
@@ -23,54 +23,54 @@ The Mixed Reality Toolkit will require Unity 2018.4.
 
 > When Unity releases an LTS (Long Term Support) product, the Mixed Reality Toolkit will update to the LTS release. MRTK will also support the latest non-beta (ex: 2019.1) tech branch version of Unity, at the time at which MRTK was released.
 
-### 2.2.0
+### 2.3.0
+
+The version 2.3.0 plan has been finalized. While some details may change as the iteration progresses, the following describes the overal plan for the next MRTK release. For the latest status of the release, please visit the [milestone page]( https://github.com/microsoft/MixedRealityToolkit-Unity/milestone/10).
 
 Status: In development
 
-Timeline: November 2019
+Timeline: January 2029
 
 Themes:
 
 - Stability
 - Developer education
-- Mobile AR
+- Unity 2019.3 XR SDK support
 - User Experience
-- Build Tools
+- Iterate on mobile AR support
 
 **Stability**
 
 Quality and stability are the top priority for this and all Microsoft Mixed Reality Toolkit releases. We will continue to prioritize customer and partner issues that impact the stability of MRTK components.
 
+The MRTK build and deploy tools remain a continued area of stability and quality focus.
+
 **Developer education**
 
 [Developer documentation](https://microsoft.github.io/MixedRealityToolkit-Unity) and example scenes are, like stability, an ongoing priority for the MRTK team.
 
-**Mobile AR**
+**Unity 2019.3 XR SDK support**
 
-The Mixed Reality Toolkit will be embracing ARFoundation from Unity to add official support for mobile AR platforms (Android ARCore and iOS ARKit). Future MRTK releases will build upon this work to support Unity's XR SDK platform.
+XR SDK is Unity's new mixed reality platform. The Microsoft Mixed Reality Toolkit is committed to supporting our customers on this new platform. All new platform support in MRTK is be delivered via XR SDK.
```

Comment:
> All new platform support

There may be times we spin up support for a platform not supported by XR SDK. Would we surface this as a custom plugin for XR SDK vs MRTK?

Reply:
good question. i'll add "is expected to be"